library/std/src/panic.rs

```diff
@@ -311,16 +311,19 @@ where
 ///
 /// # Notes
 ///
-/// Note that this function **might not catch all panics** in Rust. A panic in
-/// Rust is not always implemented via unwinding, but can be implemented by
-/// aborting the process as well. This function *only* catches unwinding panics,
-/// not those that abort the process.
+/// This function **cannot** catch panics when `panic=abort`, or with a manually written panic
+/// handler that aborts the process.
```

Comment:
We have have panics that are triggered via `panic_nounwind`. That's an internal implementation aspect, but these are panics that trigger the panic handler and yet cannot be caught, so I feel like for that case the previous text was better than the new text.

Reply:
Okay, I wasn't sure what this could be referring to other than `panic=abort`. @tgross35 asked if the original text could have "in Rust" changed to "from Rust" or "started by Rust," so I'll restore the original but with "panic(s) in Rust" changed to "Rust panics", which I think makes clear that this is specifically talking about Rust's native `panic` functionality. (I suppose I could also add the word "native" somewhere, if either of you think that would clarify things?)

Reply:
I wasn't aware of `panic_nounwind`; I think that violates my proposed wording here? https://github.com/rust-lang/reference/pull/1226/files#diff-2c4eead250247fa6e6afc3a2e118ca51da4e0eec7db7f76532e8b2abad8252e8R19-R20

Specifically:

> [panic] behavior must be uniform throughout program execution

Reply:
What I meant there was the behavior of `panic!`, `panic_any`, etc.

Reply:
"Native" would just emphasize that this isn't talking about foreign unwinding, even if there's a language Rust can link against that, like Rust, uses the word `panic` to indicate unwinding. But I don't think it's necessary or particularly helpful, so I'll stick with "Rust panics".